src/DataTypes/DataTypeDateTime.cpp

```diff
@@ -59,9 +59,20 @@ String DataTypeDateTime::doGetName() const
     return out.str();
 }
 
-void DataTypeDateTime::serializeText(const IColumn & column, size_t row_num, WriteBuffer & ostr, const FormatSettings &) const
+void DataTypeDateTime::serializeText(const IColumn & column, size_t row_num, WriteBuffer & ostr, const FormatSettings & settings) const
 {
-    writeDateTimeText(assert_cast<const ColumnType &>(column).getData()[row_num], ostr, time_zone);
+    switch (settings.date_time_output_format)
+    {
+        case FormatSettings::DateTimeOutputFormat::Simple:
+            writeDateTimeText(assert_cast<const ColumnType &>(column).getData()[row_num], ostr, time_zone);
+            return;
+        case FormatSettings::DateTimeOutputFormat::UnixTimestamp:
+            writeDateTimeTextUnixTimestamp(assert_cast<const ColumnType &>(column).getData()[row_num], ostr, time_zone);
+            return;
+        case FormatSettings::DateTimeOutputFormat::ISO:
+            /// TODO: Implement
```

Comment:
@alexey-milovidov Could you please provide some references to search in existing code base if we already have implementation for printing date in ISO format? Thanks.

Reply:
We don't have it, but it's very easy to implement.

It should print DateTime in the following form: `2020-10-11T23:42:13Z`
You can implement it similar to writeDateTimeText, but you must use UTC timezone.

Reply:
Added function in WriteHelpers.h